megamek/src/megamek/client/ui/swing/tooltip/EntityActionLog.java

```diff
@@ -0,0 +1,202 @@
+package megamek.client.ui.swing.tooltip;
```

Comment:
Please add the copyright notice to all new classes. It's in https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide

Also, as general remarks, I'll add it here because I do not want to generate a big number of change requests for rather small issues:
- please try to follow the other coding guidelines in there
- please make sure that new methods and fields, like in this class but also elsewhere, have the access type they need to have. I am not sure all of these are intentionally package-private.
- remove commented out code unless there's a really good reason to leave it (actually there's rarely. It just gets stale fast)

Reply:
will do. I thought I got rid of all commented out code, but must have missed something. which file was it in?

Reply:
Ah, I found one that snuck by in EntityActionLog, line 38. Ill keep looking.

Reply:
I added copyright, think I fixed all the accessors, etc.